From ef259184499c3a4ef1d5a90eea24069c42968042 Mon Sep 17 00:00:00 2001
From: Sebastian Walderich <sebastian@walderich.de>
Date: Mon, 23 Jul 2012 10:04:59 +0200
Subject: [PATCH] Corrected definition of wxUSE_IFILEDIALOG

The internal definition of wxUSE_IFILEDIALOG without wxDynamicLoader class
did not have any value. Because the compiler complains about empty #if
directive, the correct definition's value was added.
---
 src/msw/dirdlg.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/msw/dirdlg.cpp b/src/msw/dirdlg.cpp
index 7b29069..6b8e202 100644
--- a/src/msw/dirdlg.cpp
+++ b/src/msw/dirdlg.cpp
@@ -50,7 +50,7 @@
 #if wxUSE_DYNLIB_CLASS && !defined(__WXWINCE__)
     #define wxUSE_IFILEDIALOG 1
 #else
-    #define wxUSE_IFILEDIALOG
+    #define wxUSE_IFILEDIALOG 0
 #endif
 
 #if wxUSE_IFILEDIALOG
-- 
1.7.10.msysgit.1

